src/components/PurchaseOrder/util.js

```diff
@@ -94,12 +94,14 @@ export const getPoFieldsLabelMap = () => {
     'reEncumber': 'ui-orders.orderDetails.reEncumber',
     'metadata.createdByUserId': 'ui-orders.orderDetails.createdBy',
     'metadata.createdDate': 'ui-orders.orderDetails.createdOn',
+    'metadata.updatedByUserId': 'ui-orders.versionHistory.updatedByUserId',
     'metadata.updatedDate': 'ui-orders.order.lastUpdated',
```

Comment:
These values seem like copies of [values from stripes-components](https://github.com/folio-org/stripes-components/blob/98b90bb7b4acd5eaa50d02941fe704c4829db49d/translations/stripes-components/en.json#L82-L88) that are leveraged by the SSC [ViewMetaData component](https://github.com/folio-org/stripes-smart-components/tree/6a58c7ad181bb3c553fd9e191596af9c7a96a19a/lib/ViewMetaData). 

Is there a reason we can't use `ViewMetaData`? Even if so, is there a reason we need local translations rather than just leveraging those from `stripes-components`?

Reply:
Thanks Zak! This translation has a slightly different meaning than the one used in the `stripes-components`. It was added to provide a label for the changed key in the object, while the `stripes-components` translation points directly to the source of the change.

![image](https://user-images.githubusercontent.com/88109087/212943899-f0fc4933-820d-4b5d-9188-7932801906c2.png)